lib/chore/queues/filesystem/publisher.rb

```diff
@@ -24,6 +24,9 @@ def publish(queue_name,job)
               if f.flock(File::LOCK_EX | File::LOCK_NB) && f.size == 0
                 begin
                   f.write(job.to_json)
+                rescue StandardError => e
+                  Chore.logger.error "#{e.class.name}: #{e.message}. Could not write #{job[:class]} job to #{queue_name} file.\n#{e.stacktrace}"
+                  raise e
```

Comment:
It turns out that the exception is swallowed because we `break` in the `ensure` block below—the exception causes a jump to the ensure, but the break jumps out of the while and out of exception handling(!).

This _probably_ should have a `break` below the ensure block, but in order not to change behavior, we should remove the `raise` here.

Q_Q